includes/payment-methods/class-wc-stripe-upe-payment-method-giropay.php

```diff
@@ -18,7 +18,7 @@ class WC_Stripe_UPE_Payment_Method_Giropay extends WC_Stripe_UPE_Payment_Method
 	public function __construct() {
 		parent::__construct();
 		$this->stripe_id            = self::STRIPE_ID;
-		$this->title                = __( 'Pay with giropay', 'woocommerce-gateway-stripe' );
+		$this->title                = __( 'giropay', 'woocommerce-gateway-stripe' );
```

Comment:
```suggestion
		$this->title                = __( 'Giropay', 'woocommerce-gateway-stripe' );
```

Reply:
Believe it or not, that's the correct capitalisation for giropay. [WooPayments uses the same all lowercase form](https://github.com/Automattic/woocommerce-payments/blob/edf69a59e2a8f0913c55ba27bf69bd1a1d551d2b/includes/payment-methods/class-giropay-payment-method.php#L27). [Stripe in their documentation](https://stripe.com/docs/payments/giropay) also refer to it like this.

Reply:
My bad :smile: